xalia/Interop/Win32.cs

```diff
@@ -1024,6 +1024,9 @@ public struct MSG
         [DllImport(USER_LIB, CallingConvention = CallingConvention.Winapi)]
         public extern static bool DispatchMessageW(ref MSG lpMsg);
 
+        [DllImport(USER_LIB, CallingConvention = CallingConvention.Winapi)]
+        public extern static int RegisterWindowMessageW(string lpString);
+

```

Comment:
Probably not a big deal, but passing `string lpString` as an argument here seems to default to a BSTR from what I can tell. I've looked for other examples of using this function in C#, some add:
`[MarshalAs(UnmanagedType.LPWStr)]`
before the string arguments, and others add:
`CharSet = CharSet.Unicode`
To DllImport.

You can disregard this if it's not a big deal, just something I wanted to make sure of.

Reply:
Thanks, I'll add LPWStr.